File: NowBookingWeb.Booking/NowBookingWeb.Booking.Application/Dto/Input/Booking/CreateBookingInput.cs
namespace NowBookingWeb.Booking.Application.Dto.Input.Booking
{
    /// <summary>
    /// Класс входной модели создания бронирования.
    /// </summary>
    public class CreateBookingInput
    {
        /// <summary>
        /// Дата начала бронирования.
        /// </summary>
        public DateTime StartDate { get; set; }

        /// <summary>
        /// Дата окончания бронирования.
        /// </summary>
        public DateTime? EndDate { get; set; }

        /// <summary>
        /// Id пользователя, который бронирует.
        /// </summary>
        public long UserId { get; set; }

        /// <summary>
        /// Id категории.
        /// </summary>
        public int CategoryId { get; set; }
    }
}

File: now-booking-web-angular/src/app/booking/models/input/create-booking-input.ts
/**
 * Класс входной модели создания бронирования.
 */
export class CreateBookingInput {
  /**
   * Дата начала бронирования.
   */
  startDate: Date = new Date();

  /**
   * Дата окончания бронирования.
   */
  endDate?: Date | null = null;

  /**
   * Id пользователя, который бронирует.
   */
  userId: number = 0;

  /**
   * Id обьекта бронирования.
   */
  categoryId: number = 0;
}
